core/new-gui/src/app/workspace/component/operator-panel/operator-panel.component.ts

```diff
@@ -28,6 +38,12 @@ import { OperatorSchema, OperatorMetadata, GroupInfo } from '../../types/operato
 })
 export class OperatorPanelComponent implements OnInit {
 
+
+  myControl = new FormControl();
+  options: string[] = []
+  filteredOptions: Observable<string[]> | undefined;
```

Comment:
add scope for this variables (private or public) and rename them to make it more obvious what these variables do.

you can also add comments to describe these variables.